support-frontend/assets/helpers/redux/checkout/personalDetails/state.ts

```diff
@@ -39,6 +39,8 @@ export const personalDetailsSchema = z
 		),
 		email: emailRules,
 		confirmEmail: z.optional(z.string()),
+		// isSignedIn is maintained in personal details state purely for the email matching check
```

Comment:
Great comment, this answers my previous question about (personalDetails.isSignedIn) already!